src/mutation.cpp

```diff
@@ -46,6 +46,17 @@ bool Character::has_base_trait(const std::string &b) const
     return my_traits.find( b ) != my_traits.end();
 }
 
+void update_vitamins( Character *ch ) {
```

Comment:
dynamic_cast returns `nullptr` for a failed cast as opposed to throwing an exception